source/common/res/features/collapse-side-menu/main.js

```diff
@@ -187,9 +187,11 @@
           $('.budget-header').animate({ left: originalSizes.headerLeft });
           if ($('.budget-content').is(':visible')) {
             if (ynabToolKit.options.resizeInspector) {
-              $('.budget-content').animate({ width: ynabToolKit.resizeInspector.getContentSize(true) }, 400, 'swing', function () {
+              let bcWidth = ynabToolKit.resizeInspector.getContentSize(true);
+              $('.budget-content').animate({ width: bcWidth }, 400, 'swing', function () {
                 $('.ynabtk-navlink-collapse').removeClass('collapsed').addClass('expanded');
               });
+              $('.budget-table-container').animate({ width: bcWidth });
```

Comment:
An easier way to do this would be to just put it into the selector, yeah? So:

```
$('.budget-content, .budget-table-container').animate(...
```
on line 191, then remove this line here.

Reply:
Wouldn't that apply the function on line 191 apply to both elements? IOW, it would remove/add a class to the cudget-table-container that isn't needed. 

Also seems  I needed to do them serially to avoid a paint issue but I could be remembering that incorrectly.